.gitignore

```diff
@@ -23,6 +23,7 @@ distribute-*
 env/
 build/
 dist/
+docker/
```

Comment:
It might be better to use git-submodule for this purpose. But I'm not good at it...